x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_list.tsx

```diff
@@ -44,8 +44,8 @@ const PolicyLink: React.FC<{ name: string; route: string }> = ({ name, route })
   );
 };
 
-const renderPolicyNameLink = (value: string, _item: PolicyData) => {
-  return <PolicyLink name={value} route={`/policy/${_item.id}`} />;
+const renderPolicyNameLink = (value: string, item: Immutable<PolicyData>) => {
```

Comment:
`_` prefix in typescript bindings is used to tell `tsserver` to exclude it from unused-bindings checks. this is used